regression/arrays/standard_init1/main.c

```diff
@@ -1,7 +1,7 @@
 extern void __VERIFIER_error() __attribute__ ((__noreturn__));
 void __VERIFIER_assert(int cond) { if(!(cond)) { ERROR: __VERIFIER_error(); } }
 
-#define N 10
+#define N 1000000
```

Comment:
Was this the reason for the crashes that we have seen for large arrays? If so, we should remove the "bail out on large arrays hack" that we introduced somewhere.

Reply:
This was not the reason but (hopefully) should be a fix for that. I'll remove the hack and let's see if it works on SV-COMP benchmarks.